src/main.rs

```diff
@@ -13,8 +14,8 @@ use std::sync::Arc;
 use clap::ArgMatches;
 use env_logger::{Builder, Env, Target};
 
-fn init_logger() {
-    let env = Env::default().filter_or("NUN_LOG_LEVEL", "info");
+fn init_logger(config: &Configuration) {
+    let env = Env::default().filter_or(config.nun_log_level.as_str(), "info");
```

Comment:
Are you sure this is what you expect???  `NUN_LOG_LEVEL`  is the name of the env var. It looks like is `config.nun_log_level`  will return the value not the name of the env var if I am not mistaken.

Reply:
my bad. fixed

Reply:
my bad fixed.